config.json

```diff
@@ -244,7 +244,10 @@
     "restrict_to_login": false,
     "swatch_option_size": "22x22",
     "social_icon_placement_top": false,
-    "social_icon_placement_bottom": "bottom_none"
+    "social_icon_placement_bottom": "bottom_none",
+    "show_geotrust_ssl_seal": false,
+    "geotrust_ssl_seal_hostname": "",
```

Comment:
We do have `base_url` & `secure_base_url` for both http & https links. We also have the `home` in the `urls` section of the `context`. All the urls are with `http{s}://` so we will need to strip them out. The question is should we pass it as resource on the bcapp side or just add a helper in pencil to strip `http{s}://`.